### DISCUSSION TITLE
error in indexing text file documents with GraphRag

### DISCUSSION THREAD

**Discussion by Goodnight77** [microsoft/graphrag#540] [General]
when executing this command "python -m graphrag.index --root ." i get this error 
❌ create_base_entity_graph
None
⠹ GraphRAG Indexer
├── Loading Input (InputFileType.text) - 1 fil… …
├── create_base_text_units
├── create_base_extracted_entities
├── create_summarized_entities
└── create_base_entity_graph
❌ Errors occurred during the pipeline run, see logs for more details.

anyone get this one before ?

**Comment by Mohammad-DSAI** [comment]
hi, yes. your .txt file is too long. make it about 5 line in it and try again. please let me know if it resolved.

**Comment by Goodnight77** [comment]
hello thnx for your response, but the one file i used is less than 10 lines and it gave me that error. but the problem is that if i gave him half a book with more than 300 lines it works perfectly

**Comment by natoverse** [comment]
This may resolve your issue if it was an encoding problem: https://github.com/microsoft/graphrag/discussions/702
